pkg/gofr/container/mock_container.go

```diff
@@ -28,6 +30,14 @@ func NewMockContainer(t *testing.T) (*Container, Mocks) {
 
 	mocks := Mocks{Redis: redisMock, SQL: sqlMock}
 
+	ctrl := gomock.NewController(t)
```

Comment:
Instead of creating a new controller `gomock.NewController(t)` we can create one at the starting and pass it everywhere